build-system/tasks/presubmit-checks.js

```diff
@@ -59,6 +59,25 @@ var forbiddenTerms = {
       'test/functional/test-url-replacements.js'
     ],
   },
+  'installViewerService': {
+    message: privateServiceFactory,
+    whitelist: [
+      'src/amp-core-service.js',
+      'src/service/history-impl.js',
+      'src/service/viewer-impl.js',
+      'src/service/viewport-impl.js',
+      'test/functional/test-cid.js',
```

Comment:
not currently, i didnt since i thought that people might unintentionally whitelist something they shouldnt have. but now that the list is growing, might as well.

added PR/feature here https://github.com/ampproject/amphtml/pull/1240

/cc @cramforce